notify/impl.go

```diff
@@ -221,6 +221,13 @@ func NewEmail(c *config.EmailConfig, t *template.Template, l log.Logger) *Email
 // auth resolves a string of authentication mechanisms.
 func (n *Email) auth(mechs string) (smtp.Auth, error) {
 	username := n.conf.AuthUsername
+
+	// If no username is set, keep going without authentication
```

Comment:
missing dot at the end of the comment.

Reply:
i'm very new to Go. i will sort the formatting issues after lunch. i felt awkward about having to adapt the surrounding tests to fit this approach but its better than adding to the code smell in the Notify function. wish i had time and scope to get stuck right into it ;)